addons/mail/static/src/components/call_main_view/call_main_view.xml

```diff
@@ -4,7 +4,6 @@
 <t t-name="mail.CallMainView" owl="1">
     <t t-if="callMainView">
```

Comment:
Also note that while there is a guard in the template, `this.root.el` might actually be null.

This guard is no longer necessary, but its presence makes the new code potentially unreliable or at least hard to understand.

Reply:
Isn't it only `null` when the component shouldn't have been rendered in the first place? Can change from having no `callMainView` to having one without going through a `onMounted`?

Reply:
As long as the condition is present on the template, it can in theory be null at any time.
But as I said in practice it's never null the condition is just useless since Owl 2.
And the main flux in Owl 1 was during delete of record, which sometimes triggered render of obsolete child component before the parent component (that should remove the obsolete child).